src/server/ApiRoutes/Bib.js

```diff
@@ -232,12 +178,8 @@ function fetchBib (bibId, cb, errorcb, reqOptions, res) {
 
 function bibSearch (req, res, resolve) {
   const bibId = req.params.bibId;
-  const query = req.query;
   const { features, itemFrom } = req.query;
   const urlEnabledFeatures = extractFeatures(features);
-  let filterItemsStr = Object.keys(query)
-    .map((key) => `item_${key}=${query[key]}`)
-    .join('&');
```

Comment:
This comes in as `?status=status:a&location=loc:rm2ma` for item filtering. I think we still need this unless if there's another reason this was removed?

Reply:
weird... i didn't make that change. must have been from merge. i will look into it.